docs/dashboards/highcharts-component.md

```diff
@@ -0,0 +1,74 @@
+Highcharts Component
+===
+
+The Highcharts Component allows the end-user to define a chart in the dashboard. Charts are most often used to visualize data that changes over time.
+
+<iframe style="width: 100%; height: 470px; border: none;" src=https://www.highcharts.com/samples/embed/dashboards/components/component-highcharts allow="fullscreen"></iframe>
+
+### How to start
+To get started quickly we need to load the JavaScript and CSS files in the following order.
+
+1. To be able to use Highcharts Component you first have to load [Highcharts](https://code.highcharts.com/highcharts.js) as usual and load the additional [Dashboards plugin](https://code.highcharts.com/dashboards/modules/dashboards-plugin.js).
+
+```html
+<script src="https://code.highcharts.com/dashboards/dashboards.js"></script>
+<script src="https://code.highcharts.com/highcharts.js"></script>
+<script src="https://code.highcharts.com/dashboards/modules/dashboards-plugin.js"></script>
+```
+
+2. The Highcharts Component uses [styledMode](https://api.highcharts.com/highcharts/chart.styledMode) by default, so you need to load also the set of CSS styles to display Highcharts properly.
+```css
+@import url("https://code.highcharts.com/css/highcharts.css");
+```
+More information about styling charts, you can find in our [docs](https://www.highcharts.com/docs/chart-design-and-style/style-by-css).
+
+3. Now you have loaded all required files, so you should define the cell `cell: 'dashboard-col-0'`, where the component should be created.
```

Comment:
It doesn't have to be `dashboard-col-0'. I would rephrase it like: 

`After loading the necessary files, define a cell using a unique identifier for  example cell: 'dashboard-col-0'`